java/core/src/java/org/apache/orc/impl/TreeReaderFactory.java

```diff
@@ -201,6 +201,10 @@ public void nextVector(ColumnVector previous,
     public BitFieldReader getPresent() {
       return present;
     }
+
```

Comment:
What do you think about moving the building of writerIncluded into the SchemaEvolution class? I think it would be pretty natural there and building it would come naturally out of buildMapping.